test/e2e/tests/security-provider.spec.js

```diff
@@ -0,0 +1,255 @@
+const { strict: assert } = require('assert');
+const { convertToHexValue, withFixtures } = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+const OPENSEA_URL =
+  'https://eos9d7dmfj.execute-api.us-east-1.amazonaws.com/metamask/validate';
+/**
+ * @param {import('mockttp').Mockttp} mockServer - The mock server.
+ */
+
+describe('Transaction security provider', function () {
+  let windowHandles;
+  // eslint-disable-next-line no-unused-vars
+  let extension;
```

Comment:
I think we can remove this line as it's not needed anymore

Reply:
Done